### PR TITLE
Add Repo-rater badge

### PR DESCRIPTION
**Pull request by dipanshurdev**
Added Repo-rater badge to README.md

screenshot:
Screenshot 2024-01-16 110149

### CODE REVIEW

**Comment by CalvinWilkinson** [comment]
Hello @amanr-dev!!

Thanks for your contribution!! 

### **Just a quick heads up though.**

When contributing to the project, (or other people's projects), please make sure that you follow the contribution guidelines for that project.  This was not an official PR and there is no issue for this PR using one of the organizational issue templates.  Also, the branch naming is all wrong as well.  Due to it being a simple change and your first contribution, I will let this one through. 

But just to let you know, if you want to make any other contributions, I will have to reject the pull request.

### **Why is this?**
Not every project is the same.  Each project's either has its requirements on how it is run which is forced by the tech stack that the maintainers are using, and/or it might be just because of non-required choices made within the team.  Some of the reasons why it is important to follow contribution guidelines are because the development workflow for a project might be required to make sure that security practices are in check, branch naming standards and git workflows are enforced, development workflows are kept clear, consistent, and concise, ensuring execution of CICD workflows and status checks, and more.  

### **Why am I letting this PR through then?**
I believe in being inclusive and educating any interested contributors and I don't want them to feel unwelcome.
I have done my part to make sure you know the requirements for following the contribution guidelines for the health of the project.

> [!Note]
> Please don't take this the wrong way!!  No harm has been done!!

### **Here are the reasons I am merging this PR:**
1. You took time out of your day to contribute something, and that also means something
2. I want to make sure you feel welcome in the community.
3. I was gonna add this badge anyway, you just beat me to it!!  😀
4. Maybe you are new to development and are unaware of these kinds of things.  That would not be your fault and I don't intend to make you feel like it is.  😉
5. I know Eddie Jaoude (the creator/maintainer of Repo Rator) and he is a friend of mine, so this supports him.  🤘🏻

All is good!!  If you are interested in joining our [Discord Server](https://discord.gg/qewu6fNgv7), hop in and I can answer any questions you have.
You can also read the contribution guidelines [here](https://github.com/KinsonDigital/.github/blob/main/docs/CONTRIBUTING.md).  These guidelines have best practices, git expectations with branching, and more.

File: README.md
Comment (CalvinWilkinson) [inline README.md]: Please add a single empty line to this spot.

**Reason:**
I have some special processing going on with the readme file in my CICD system.
I don't think it will break anything to delete this line, but I would rather not take the chance.

File: README.md
Comment (CalvinWilkinson) [inline README.md]: Please add a single empty line to this spot.

**Reason:**
I have some special processing going on with the readme file in my CICD system.
I don't think it will break anything to delete this line, but I would rather not take the chance.

File: README.md
Comment (CalvinWilkinson) [inline README.md]: Please add a single empty line to this spot.

**Reason:**
I have some special processing going on with the readme file in my CICD system.
I don't think it will break anything to delete this line, but I would rather not take the chance.

File: README.md
Comment (CalvinWilkinson) [inline README.md]: Please add a single empty line to this spot.

**Reason:**
I have some special processing going on with the readme file in my CICD system.
I don't think it will break anything to delete this line, but I would rather not take the chance.

**Comment by CalvinWilkinson** [comment]
@amanr-dev If you could do me a favor, and move the badge just to the right of the **_Code Coverage_** badge, that would be great.

Once this is done, I will  merge it.
image

**Review by CalvinWilkinson — CHANGES_REQUESTED** [review]

File: README.md
Comment (dipanshurdev) [reply README.md]: After line number 8?

**Comment by dipanshurdev** [comment]
> @amanr-dev If you could do me a favor, and move the badge just to the right of the **_Code Coverage_** badge, that would be great.
> 
> Once this is done, I will merge it. image

Sure.

**Comment by CalvinWilkinson** [comment]
> > @amanr-dev If you could do me a favor, and move the badge just to the right of the **_Code Coverage_** badge, that would be great.
> > Once this is done, I will merge it. image
> 
> Sure.

Also, please don't hesitate to let me know when it is ready for me to check it out.  Since the pull request was not created in draft mode, there is not way to see if the PR is ready because it was never in draft.   I will wait for a github notification to find out that you have commented on the pr and that it is ready for review.

**Comment by dipanshurdev** [comment]
> > > @amanr-dev If you could do me a favor, and move the badge just to the right of the **_Code Coverage_** badge, that would be great.
> > > Once this is done, I will merge it. image
> > 
> > 
> > Sure.
> 
> Also, please don't hesitate to let me know when it is ready for me to check it out. Since the pull request was not created in draft mode, there is not way to see if the PR is ready because it was never in draft. I will wait for a github notification to find out that you have commented on the pr and that it is ready for review.

Hi @CalvinWilkinson, I added space to the required lines in README.md. Now you can review it.

**Comment by dipanshurdev** [comment]
> > > > @amanr-dev If you could do me a favor, and move the badge just to the right of the **_Code Coverage_** badge, that would be great.
> > > > Once this is done, I will merge it. image
> > > 
> > > 
> > > Sure.
> > 
> > 
> > Also, please don't hesitate to let me know when it is ready for me to check it out. Since the pull request was not created in draft mode, there is not way to see if the PR is ready because it was never in draft. I will wait for a github notification to find out that you have commented on the pr and that it is ready for review.
> 
> Hi @CalvinWilkinson, I added space to the required lines in README.md. Now you can review it.

And I also moved the `repo-rater` badge to the right side of `code coverage`

**Comment by CalvinWilkinson** [comment]
> > > > @amanr-dev If you could do me a favor, and move the badge just to the right of the **_Code Coverage_** badge, that would be great.
> > > > Once this is done, I will merge it. image
> > > 
> > > 
> > > Sure.
> > 
> > 
> > Also, please don't hesitate to let me know when it is ready for me to check it out. Since the pull request was not created in draft mode, there is not way to see if the PR is ready because it was never in draft. I will wait for a github notification to find out that you have commented on the pr and that it is ready for review.
> 
> Hi @CalvinWilkinson, I added space to the required lines in README.md. Now you can review it.

Thanks @amanr-dev!!  I will review it out this morning.

**Review by CalvinWilkinson — APPROVED** [review]

**Pull request merged**
